docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md

```diff
@@ -38,13 +38,13 @@ host all yugabyte 127.0.0.1/0 password,"host all all 0.0.0.0/0 ldap ldapserver=l
 
 For more information, see [Edit configuration flags](../../../yugabyte-platform/manage-deployments/edit-config-flags/).
 
-When entering the flag value in YugabyteDB Anywhere, do not enclose it in single quotes, as you would in a Linux shell.
+When entering the flag value in YugabyteDB Anywhere, **do not** enclose it in single quotes, as you would in a Linux shell.
```

Comment:
We don't use bold font randomly. It is reserved for emphasizing UI elements related to a screenshot. So, in this case:

do not enclose

Reply:
Maybe we can add an example that clearly shows that there are no quotes